include/dsAudio.h

```diff
@@ -1142,78 +1074,15 @@ dsError_t  dsGetAudioLevel(intptr_t handle, float *level);
- * @retval dsERR_GENERAL                  -  Underlying undefined platform error
- * 
- * @pre  dsAudioPortInit() and dsGetAudioPort() should be called before calling this API.
- * 
- * @warning  This API is Not thread safe.
- */
-dsError_t  dsGetAudioMaxDB(intptr_t handle, float *maxDb);
-
-/**
- * @brief Gets the minimum audio dB level of an audio port.
- * 
- * This function returns the minimum audio dB level supported by the audio port corresponding to specified port handle.
- *
- * @param[in] handle  - Handle for the output audio port
- * @param[out] minDb  - Pointer to hold the minimum audio dB value (float. e.g: 0.0) supported by the specified audio port(platform specific)
- *                        Minimum value can be -1450 dB
- *
- * @return dsError_t                      -  Status 
- * @retval dsERR_NONE                     -  Success
- * @retval dsERR_NOT_INITIALIZED          -  Module is not initialised
- * @retval dsERR_INVALID_PARAM            -  Parameter passed to this function is invalid
- * @retval dsERR_OPERATION_NOT_SUPPORTED  -  The attempted operation is not supported
- * @retval dsERR_GENERAL                  -  Underlying undefined platform error
- * 
- * @pre  dsAudioPortInit() and dsGetAudioPort() should be called before calling this API.
- * 
- * @warning  This API is Not thread safe.
- */
-dsError_t  dsGetAudioMinDB(intptr_t handle, float *minDb);
-
-/**
- * @brief Gets the optimal audio level of an audio port.
- * 
- * This function returns the optimal audio level (dB) of the audio port corresponding to specified port handle(platform specific).
- *
- * @param[in] handle        - Handle for the output audio port
- * @param[out] optimalLevel - Pointer to hold the optimal level value of the specified audio port(platform specific)
- *
- * @return dsError_t                      -  Status 
- * @retval dsERR_NONE                     -  Success
- * @retval dsERR_NOT_INITIALIZED          -  Module is not initialised
- * @retval dsERR_INVALID_PARAM            -  Parameter passed to this function is invalid
- * @retval dsERR_OPERATION_NOT_SUPPORTED  -  The attempted operation is not supported
- * @retval dsERR_GENERAL                  -  Underlying undefined platform error
- * 
- * @pre  dsAudioPortInit() and dsGetAudioPort() should be called before calling this API.
- * 
- * @warning  This API is Not thread safe.
- */
-dsError_t  dsGetAudioOptimalLevel(intptr_t handle, float *optimalLevel);
-
 /**
  * @brief Gets the audio delay (in ms) of an audio port
  *
- * This function returns the audio delay (in milliseconds) of audio port with respect to video corresponding to the specified port handle.
+ * For sink devices, this function returns the digital audio delay (in milliseconds) of the digital interfaces(HDMI ARC/eARC, SPDIF).
+ * The Audio delay ranges from 0 to 200 milliseconds.
+ * For source devices, this function returns the digital audio delay (in milliseconds) of the digital interfaces(HDMI)
```

Comment:
Don't we have SPDIF on STBs?

Reply:
SPDIF is supported in one STB

Reply:
Do we need to have this open? If the question is clarified, let's resolve.

Reply:
Resolved